docs/site/sidebars/lb4_sidebar.yml

```
@@ -264,6 +264,20 @@ children:
           url: Using-openapi-to-graphql.html
           output: 'web, pdf'
 
+    - title: 'Creating Artifacts at Runtime'
+      url: Creating-artifacts-runtime.html
+      output: 'web, pdf'
+      children:
+         - title: 'Creating Model at Runtime'
+           url: Creating-model-runtime.html
+           output: 'web, pdf'
+         - title: 'Creating DataSource at Runtime'
+           url: Creating-datasource-runtime.html
+           output: 'web, pdf'
+         - title: 'Creating Repository at Runtime'
```

I think the titles here should match their files like:
```suggestion
         - title: 'Creating Repositories at Runtime'
```